client/src/components/teamMemeber/index.js

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
 import Paragraph from '../paragraph';
+import './teamMember.css';
+import { Card, Col } from 'react-bootstrap';
 
-export default function TeamMember(props){
-    let skills = props.skills.map(skill => {
-       return <li>{skill.name} {skill.years}</li>
-    })
- 
-    return(
-        <div class="teamMember" id={props.id}>
-            <img src={props.img}></img>
-           <header>{props.name}</header>
-           <br></br>
-           <ul>
+export default function TeamMember(props) {
+    // let skills = props.skills.map(skill => {
```

Comment:
Are you still deciding if you will need this? If you know you won't need it, please remove (the comments)!

Reply:
This is should print out a list of skills but right now it is not working as expected,  I plan to get it to work at a later time but it is not a priority --I will add a comment with that explanation :)